### PR TITLE
8.0 base format

### PR DESCRIPTION
**Pull request by foutoucour**
This module allows to manage dimensions.
Specifics features for Book Industry : certain format of book (lxL) by country, perhaps usable for other activity.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
_base_format_ doesn't seem to be a good name for the module, because it's too generic.

**Comment by foutoucour** [comment]
@pedrobaeza would you have a name to suggest?
Maybe something like base_dimension_format or base_edition_format?

File: base_format/base_format.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+class base_format(orm.Model):
+    """ This model stores dimensions."""
+    _name = "base.format"
+    _description = "This model stores dimensions."
+
+    _columns = {
+        'name': fields.char('Name', required=True, translate=True, select=True,
+                            size=64),
+        'format_uom': fields.many2one('product.uom', string='Format UoM'),
```
Comment (pedrobaeza) [inline base_format/base_format.py:37]: Why not only uom? You are adding this field to format model, so you know that it's the UoM of the format.

File: base_format/base_format.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        "width": fields.float('Width'),
```
Comment (pedrobaeza) [inline base_format/base_format.py:38]: You don't need to put the labels if it is the same as the field name capitalized.

File: base_format/base_format.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        "height": fields.float('Height'),
+        "comment": fields.text('Comment'),
+        "active": fields.boolean('Active'),
+    }
+
+    _sql_constraints = [
+        ("name_unique", "unique(name)",
+         _("Can't be duplicate value for the field 'Name'!")),
```
Comment (pedrobaeza) [inline base_format/base_format.py:46]: `There can't be a duplicated value for the field 'Name'!`

**Comment by pedrobaeza** [comment]
Check also some PEP8 and pylint errors.

About the name, I'm thinking something like _publication_paper_formats_, in the same line of the _publication_management_ module also published. This is not generic enough for being `base_...` I think.

**Comment by pedrobaeza** [comment]
As there's no answer for comments since a lot, I close. If there's interest, please reopen.

**Pull request closed without merging**